pkgs/applications/audio/ossia-score/default.nix

```diff
@@ -0,0 +1,113 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+, qttools
+, wrapQtAppsHook
+, alsa-lib
+, boost
+, faust
+, ffmpeg
+, fftw
+, flac
+, lame
+# , libclang
+, libjack2
+, libopus
+, libvorbis
+, lilv
+# , llvm
+, lv2
+, mpg123
+, portaudio
+, portmidi
+, qtbase
+, qtdeclarative
+, qtquickcontrols2
+, qtserialport
+# TODO: not part of Qt5?
+# , qtshadertools
+, qtsvg
+, qtwebsockets
+, suil
```

Comment:
```suggestion
, suil-qt5
```

Reply:
I'm not seeing the `suil-qt5` package anymore. Was it removed?